dataikuapi/dss/notebook.py

```diff
@@ -9,8 +10,10 @@ def __init__(self, client, project_key, notebook_name, state=None, content=None)
        self.project_key = project_key
        self.notebook_name = notebook_name
        self.state = state
-       self.content = content
-       self.state_is_peek = True
+       if content:
+           self.content = content
+       else:
+           self.content = DSSNotebookContent(self.client, self.project_key, self.notebook_name)
```

Comment:
```suggestion
      self.content = content
```
Not sure if it makes sense to create a DSSNotebookContent object with no content 
I think maybe `get_content()` should do the api call to get the content and return a DSSNotebookContent object like we do for other DSSObjects, WDYT ?

Reply:
Since the NotebookContent currently contains only the metadata to retrieve the content, I'm fine with my implementation.
But I don't have a strong feeling about this part.
It may evolve if we decide that the DSSNotebookContent must contains the content.